src/g1.rs

```diff
@@ -417,6 +430,7 @@ impl G1Affine {
 
 /// This is an element of $\mathbb{G}_1$ represented in the projective coordinate space.
 #[cfg_attr(docsrs, doc(cfg(feature = "groups")))]
+#[cfg_attr(feature = "zeroize", derive(Zeroize))]
```

Comment:
Ditto. Remove this:
```suggestion
```

and instead, right after the `impl Default for G1Projective` (which returns the identity), add:
```rust
#[cfg(feature = "zeroize")]
impl zeroize::DefaultIsZeroes for G1Projective {}
```